Matricies/Matricies/Measurments.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using System.Threading.Tasks;
+using System.Timers;
+using System.Diagnostics;
+
+
+namespace Matricies
+{
+    public class Measurments
+    {
+            int numOfIterations;
+
+            string res_path;
+
+            string f_path;
+            string s_path;
+
+            public Measurments(int numIt, string path, string path_1, string path_2)
+            {
+                numOfIterations = numIt;
+
+                f_path = path_1;
+                s_path = path_2;
+                res_path = path;
+            }
+
+        /// <summary>
+        /// counts time of work of a given method 
+        /// </summary>
+        
+            long count_time(MyMatrix A, MyMatrix B, Func<MyMatrix, MyMatrix, MyMatrix> func)
+            {
+                Stopwatch timer = new Stopwatch();
```

Comment:
```suggestion
                var timer = new Stopwatch();
```
Не пишите один и тот же тип на одной строке дважды